app/assets/javascripts/admin/components/theme-translation.js.es6

```diff
@@ -0,0 +1,21 @@
+import BufferedContent from "discourse/mixins/buffered-content";
+import SettingComponent from "admin/mixins/setting-component";
+import computed from "ember-addons/ember-computed-decorators";
+
+export default Ember.Component.extend(BufferedContent, SettingComponent, {
+  layoutName: "admin/templates/components/site-setting",
+  setting: Ember.computed.alias("translation"),
+
+  @computed("translation.key")
+  settingName(key) {
+    return key;
+  },
+
+  type: "string",
```

Comment:
Bit weird for this variable to be initialized down here. I'd suggest moving it up top.